packages/api-client/src/connections/EntityApi.ts

```diff
@@ -101,14 +99,10 @@ type Prefix<T, P extends string> = {
   [field in keyof T & string as `${P}_${field}`]: T[field];
 };
 
-export class EntityApi implements MicroserviceApi {
-  public baseUrl = ENTITY_API_URL;
```

Comment:
Externalised this config variable. 

Previously this meant that EntityApi needs to know to look on process.env, however, the api client may be configured in a different way. E.g. in tamanu they don't use env vars. 

New implementation has the TupaiaApiClient responsible for picking good default base urls, and these can be overridden by passing in custom ones, e.g. for feature branches.